src/cli/domain/watch.js

```diff
@@ -14,8 +14,9 @@ module.exports = function(dirs, baseDir, changed) {
       (fileName, currentStat, previousStat) => {
         if (!!currentStat || !!previousStat) {
           if (
-            /node_modules|package.tar.gz|_package|\.sw[op]/gi.test(fileName) ===
-            false
+            /node_modules|package\.tar\.gz|_package|\.sw[op]|\.git/gi.test(
+              fileName
+            ) === false
```

Comment:
I prefer this prettified version over the original since it puts the `===` on the same line as the `false`.